conans/test/integration/configuration/default_profile_test.py

```diff
@@ -180,3 +181,19 @@ def build(self):
                           "an existing profile file.", client.out)
 
 
+def test_conf_default_two_profiles():
+    client = TestClient()
+    save(os.path.join(client.cache.profiles_path, "mydefault"), "[settings]\nos=FreeBSD")
+    save(os.path.join(client.cache.profiles_path, "mydefault_build"), "[settings]\nos=Android")
+    global_conf = textwrap.dedent("""
+        core:default_profile=mydefault
```

Comment:
Just one question, why is the name `default_profile` instead of `default_host_profile`

Reply:
To keep it a bit more aligned with the current naming, documentation, etc.